test/test_sentry_processor.py

```diff
@@ -236,3 +237,41 @@ def test_sentry_ignore_logger(sentry_events, level):
     assert_event_dict(event_data, sentry_events)
     assert blacklisted_logger_event_dict.get("sentry") == "ignored"
     assert whitelisted_logger_event_dict.get("sentry") != "ignored"
+
+
+def test_sentry_json_respects_global_with_locals_option_no_locals():
+    with unittest.mock.patch.object(SentryProcessor, "_get_hub") as mock__get_hub:
```

Comment:
Could you please use `sentry_events` fixture instead? It does not require mocking and it contains captured sentry events. For inspiration see other tests.